src/utils/schemaPrinter.ts

```diff
@@ -106,56 +121,15 @@ export function printSchemaComposer(
   sc: SchemaComposer<any>,
   options?: SchemaComposerPrinterOptions
 ): string {
-  const { exclude = [], include, omitDirectiveDefinitions, ...innerOpts } = options || {};
-
-  const includeTypes = new Set();
-  if (Array.isArray(include) && include.length) {
-    include.forEach((s) => {
-      if (s && typeof s === 'string') {
-        includeTypes.add(sc.getAnyTC(s));
-      }
-    });
-  } else {
-    if (sc.has('Query')) includeTypes.add(sc.getOTC('Query'));
-    if (sc.has('Mutation')) includeTypes.add(sc.getOTC('Mutation'));
-    if (sc.has('Subscription')) includeTypes.add(sc.getOTC('Subscription'));
-  }
+  const { optPrinter, optFilter } = splitOptionsFilterPrinter(options);
 
-  const res = [];
-  if (!omitDirectiveDefinitions) {
-    const directives = sc._directives.filter((d) => !BUILT_IN_DIRECTIVES.includes(d));
-    res.push(...directives.map((d) => printDirective(d, innerOpts)));
```

Comment:
@nodkz You're right, I didn't, I only ran `npx jest SchemaComposer-test.ts` because my machine is slow, but after I finished I should have run them all.

I just pushed another commit fixing them, the tests passed on my machine this time (there are still some linting errors I didn't fix yet), can you review it ?